CHANGELOG.md

```diff
@@ -4,6 +4,31 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/#semantic-versioning-200).
 
+## [1.2.0] - 2025-03-01
+
+### :magic_wand: Added
+
+- Enhanced Failover Monitoring 2 Plugin ([PR #402](https://github.com/aws/aws-advanced-nodejs-wrapper/pull/369)).
+- Custom Endpoint Plugin ([PR #369](https://github.com/aws/aws-advanced-nodejs-wrapper/pull/369)).
+- Handling for Multi-AZ clusters to Failover 2 ([PR #396](https://github.com/aws/aws-advanced-nodejs-wrapper/pull/396)).
+- Failover2 Plugin for more aggressive failover response ([PR #363](https://github.com/aws/aws-advanced-nodejs-wrapper/pull/363)).
+- Benchmarks for the PluginManager.([PR #330](https://github.com/aws/aws-advanced-nodejs-wrapper/pull/330)).
+- Documentation on session state, driver dialect, fastest response strategy, failover2, and maintenance ([PR #350](https://github.com/aws/aws-advanced-nodejs-wrapper/pull/350)), ([PR #351](https://github.com/aws/aws-advanced-nodejs-wrapper/pull/351)), ([PR #362](https://github.com/aws/aws-advanced-nodejs-wrapper/pull/362)) ([PR #378](https://github.com/aws/aws-advanced-nodejs-wrapper/pull/378)) & ([PR #403](https://github.com/aws/aws-advanced-nodejs-wrapper/pull/403)) .
```

Comment:
Since this is for documentation can we link to the doc directly instead of the PRs